requirements/common.txt

```diff
@@ -12,7 +12,7 @@ drfdocs==0.0.11
 pika==0.10.0
 pickleshare==0.7.4
 Pillow==3.4.2
-psycopg2==2.6.2
```

Comment:
@16116073 The screenshot you pasted isn't in synchronization with this code change. Also, please revert this change as it is not the part of this pull request.

Reply:
i faced this issue while installing, 
https://github.com/psycopg/psycopg2/issues/594
After upgrading my postgres version and updating my requirements/common.txt  file made it work for me.

Reply:
Okay. So, I think this should be the part of the another pull request and this pull request should contain the changes relevant to this issue. Please make this change in another pull request as need to maintain the modularity in the code.